src/theory/sets/theory_sets_private.cpp

```diff
@@ -672,123 +387,132 @@ void TheorySetsPrivate::fullEffortCheck(){
       }
     }
 
-    flushLemmas( lemmas );
```

Comment:
Why isn't this line replaced with `d_im.flushPendingLemmas`?

Reply:
We now insist that lemmas are flushed locally by the modules that generate them.  See e.g. https://github.com/CVC4/CVC4/pull/3085/files#diff-afb263928d078d3259ca8f32785fa402R177